ortho_seq_code/orthogonal_polynomial.py

```diff
@@ -64,16 +65,60 @@ def orthogonal_polynomial(filename, phenotype, sites, dm, pop_size, out_dir):
     # ------------Converting letters to vectors---------------
     # phi[individual][site][state]. phi[i][j] = vector for site j
     # in individual i.
-    for i in range(pop_size):  # individual
-        for j in range(sites):
-            if seq[i][j] == 'A':
-                phi[j][i][0] = 1.0
-            if seq[i][j] == 'C':
-                phi[j][i][1] = 1.0
-            if seq[i][j] == 'G':
-                phi[j][i][2] = 1.0
-            if seq[i][j] == 'T':
-                phi[j][i][3] = 1.0
+    if molecule == 'DNA':
+        for i in range(pop_size):  # individual
+            for j in range(sites):
```

Comment:
declare a list/constant called  DNA_ALPHABETS  = ['A', 'C', 'G', 'T'] at the top of the program
and then use that list as below

for dna_alphabet_index in range(len(DNA_ALPHABETS)):
      if seq[i][j] == DNA_ALPHABETS[dna_alphabet_index]:
          phi[j][i][dna_alphabet_index] = 1.0

and you could do the same for the below with the amino acid alphabets

does that make sense?